linux/LinuxProcessList.c

```diff
@@ -796,6 +805,36 @@ static bool LinuxProcessList_readCmdlineFile(Process* process, const char* dirna
    process->basenameOffset = tokenEnd;
    setCommand(process, command, lastChar + 1);
 
+   LinuxProcess *lp = (LinuxProcess *)process;
+   lp->procCmdlineBasenameOffset = tokenStart;
+
+   /* /proc/[pid]/comm could change, so should be udpated */
+   if (lp->procComm)
```

Comment:
free(NULL) is a NOP

Reply:
Yes. But I have seen coding conventions insisting upon NULL check before free maybe to satisfy some old memory checkers. In htop codebase I see that except for some places like AffinityPanel.c:45 this convention is not followed. So I can remove the NULL check in the next commit.